roiextractors/segmentationextractor.py

```diff
@@ -265,13 +265,13 @@ def get_num_planes(self):
         return self._num_planes
 
     @staticmethod
-    def write_segmentation(segmentation_extractor, savepath):
+    def write_segmentation(segmentation_extractor, save_path, metadata=None, file_overwrite=False):
```

Comment:
the base extractor should only have `segmentation_extractor` and `save_path`. Additional args can be added in derived classes